ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -104,6 +104,15 @@ tests:
   commands: make test-unit
   container:
     from: src
+- always_run: false
+  as: e2e-agent-compact-ipv4
```

Comment:
Shall we not use compact ipv4 with add nodes instead?

Reply:
This PR is just for adding the standard compact job in the MCO repo (but it will get it automatically in future)